code/modules/vore/eating/digest_act_vr.dm

```diff
@@ -58,6 +58,8 @@
 	return FALSE
 /obj/item/device/perfect_tele_beacon/digest_act(var/atom/movable/item_storage = null)
 	return FALSE //Sorta important to not digest your own beacons.
+/obj/item/organ/internal/brain/slime/digest_act(var/atom/movable/item_storage = null)
+	return FALSE //so prometheans can be recovered
```

Comment:
Kinda wish this were a toggle or something. I know in the past when things like posibrains have come up, there have been two distinct sides to the debate: those that wanna gurgle as a synth and those that don't wanna. That's do-able for synths by picking different brain types...

Reply:
A toggle'd be good but I'm not sure what a good toggle would be. In order to end up as a core in a stomach in the first place you'd need to have digestion turned on.

Reply:
Yeah but it removes the ability for prometheans to be fully digested. Like if someone said they were gonna do that with FBPs I'd bap them because it's bad to just decide for everyone that you can't gurg anymore.

I would suggest the following more comprehensive solution to "people that can be sorta digested but not fully digested":
1) Make a new trait (some workshopping on a name and description may be required) that in the apply() searches for the 'brain stuff' you want to gurgle, or not
2) Add a var on the 'brain stuff' for digestable
3) Have digest_act do a thing based on that var (or return false or whatever it does)

The thing is I'm not sure what needs to be done for MMIs for this because mmi/posi/digital brains are:
Outside) An /obj/item/organ/mmi_holder thing that holds a
Layer 1) /obj/item/weapon/whatever/mmi actual mmi object that holds a
Layer 2) /mob/living/carbon/brain that the player actually inhabits when necessary

I want to say there's a special thing in the digestion code somewhere that allows the deletion of the organ mmi holder and moves the posibrain up a level? So pay attention to that.

Reply:
So far, all I can find relating to that is posibrains (a child of mmis) return false on the proc, and a comment in there indicating that there's a TODO.

Reply:
That's fine. That object that gets taken and put into guts safely is the one to add a new var on, along with promethean cores.